### ISSUE TITLE
False positive eslint-config-prettier eslint-plugin-prettier

### ISSUE THREAD

**Issue by VincentLanglet** [depcheck/depcheck#316]
### Bug Description
Unused devDependencies

### Code snippets
Used in the package json.
```
"eslintConfig": {
    "extends": [
      "react-app",
      "plugin:prettier/recommended"
    ]
  },
```

### Versions
 - `node -v`: 11.6.0
 - `npm -v`: 6.5.0-next.0
 - `depcheck --version`: 0.7.1

**Comment by cyrilchapon** [comment]
I think I found why.

Specifying the eslint config inside package.json `esLintConfig` leads to false positive, whereas using a .eslintrc file with exact same config seems to trigger the special mode "eslint" which ignores those files

**Comment by cyrilchapon** [comment]
Exact same additional bug : the package.json `"parser": "babel-eslint"` (be it at the root of package.json or inside `esLintConfig` leads to `babel-eslint` being reported.

This is not the case when set inside .eslintrc file
